Libraries/LibJS/Bytecode/Instructions.h

```diff
@@ -0,0 +1,218 @@
+/*
+ * Copyright (c) 2025, Leon Albrecht <leon.a@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "Instruction.h"
```

Comment:
first, `#include <LibJS/Bytecode/Instruction.h>`
second, really not a nice name - Instructions.h vs Instruction.h is just confusing.

Reply:
Fair point, will think of a better name